hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/PathsTraverser.java

```diff
@@ -124,9 +124,6 @@ public void forward(Id targetV, Directions direction) {
                     PathSet results = this.record.findPath(target, null,
                                                            true, false);
                     for (Path path : results) {
-                        if (Objects.equals(target, targetV)) {
```

Comment:
maybe someone don't want a ring path? can add a optional param

Reply:
it will remove the exist path due to the skip logic, a->b->c
when we first traverser from a to c, we'll bypass c then lost the only one path

Reply:
not sure what it means, can add more details?

Reply:
the `targetV` will be the same with `target`, it means when we traverser from `a` to `c`, the first time we arrived c, we drop the path directly. (details in the context)